doc/charm++/manual.rst

```diff
@@ -5863,9 +5863,25 @@ is ``unsigned short``, limiting each PE to 65,535 outstanding futures.
 To increase this limit, build Charm++ with a larger *CMK_REFNUM_TYPE*, e.g. specifying
 ``--with-refnum-type=uint`` to use ``unsigned int`` when building Charm++.
 
-
-The Converse version of future functions can be found in the :ref:`conv-futures`
-section.
+There are additional facilities for operating on collections of futures, which include:
+
++-------------------+-----------+-------------------------------------------------+
```

Comment:
Two quick questions. This will increase the character-width of the table, is it OK if I exceed the 80 character (de jure) limit? Also, they include template type arguments. Do you think it would be OK to list them as such (without context),  `std::pair<T, InputIter>`, `std::vector<T>`, and `std::pair<std::vector<T>, InputIter>`,  or should I also include descriptions of `T` and `InputIter` here?